route/TAS.py

```diff
@@ -12,6 +12,7 @@
 
 from engine.seq import SeqList, SeqLog, SequencerEngine
 from GUI import Window
+from GUI.GUI import LayoutHelper
```

Comment:
Better to add `LayoutHelper` to the GUI `__init__.py`, so it can be imported on the line above.